audmath/core/api.py

```diff
@@ -281,6 +322,14 @@ def to_nanos(value, unit):
     elif duration.__class__.__name__ == 'Timedelta':
         duration = duration.total_seconds()
 
+    # handle nan/none durations
+    elif (
+            duration is None
+            or duration.__class__.__name__ == 'NaTType'
```

Comment:
In case we also want to support `pd.NA` we need to extend the test to:

```suggestion
            or duration.__class__.__name__ == 'NaTType'
            or duration.__class__.__name__ == 'NAType'
```

Currently, we are getting are rather strange error:

```python
>>> audmath.duration_in_seconds(pd.NA)
TypeError: boolean value of NA is ambiguous
```

Reply:
I added support for `pd.NA` now, mention it in the docstring, and added a two tests for it.